source/draw/DrawContext.ooc

```diff
@@ -39,9 +39,10 @@ DrawContext: abstract class {
 	createYuv420Semiplanar: abstract func ~fromImages (y, uv: Image) -> Image
 	createYuv420Semiplanar: abstract func ~fromRaster (raster: RasterYuv420Semiplanar) -> Image
 	createImageFromTextAndFont: virtual func (message: Text, fontAtlas: Image) -> Image {
-		target := this createMonochrome(This getTextPixelBounds(message, fontAtlas))
+		target := this createMonochrome(This getTextPixelBounds(message take(), fontAtlas))
```

Comment:
I did that below to avoid writing `message take()[i]` but that takes an extra line in each little function.